### DISCUSSION TITLE
2023-11-11

### DISCUSSION THREAD

**Discussion by hanseulhee** [10-rolling/rollingPolling#59] [Daily_Meeting]
### 📝 회의내용

**- 오늘의 tmi**
수님 : 누워있다 ...? 술을 또 드셔따 .. 술꾼! 남궁술님 ㅎㅎ
수영님 : 오늘 19시에 케이크 픽업할 예정 ... 사랑꾼 (서프라이즈가 아니다)
윤혁님 : 배고파 쓰러짐 .. 누구땜에 밥을 못먹음 .. (남궁술님), 주방가서 있는 거 ..
슬희님 : 오늘은 하루종일 ... 개발한다 (목표!)

**- 멘토링 고쳐야할 사항들**
폰트 선택 시 내려가지 않음
카드가 바로 안불러와지고 새로 고침 시 불러와짐
editor 이상한 거 껴서 옴 (띄어쓰기 = &nbsp;)
id 트리거가 안됌 (원인 파악 완료)
폰트 추가
post page 이미지 찌그러짐 (배율 설정)
post page 페이지가 너무 큼 (스크롤 ㄴ ㄴ)
post page에 header 추가해라

> 수영님 : 기능 다하고 하겠다. 
수님 : 제가 폰트 추가만 !!!!! 하겠다.

**- 각자 한 거 발표하기**
수님 : 수영님 오류 잡는 거 도왔음 (2시간 넘게 ...)
수영님 : 무한 스크롤 빼고 기능해서 머지 완, 다시 pr 올려놈, 무한스크롤 (y) 해야됌 ....
윤혁님 : 수영님 오류 잡는 거 도왔음, 모달창 기능 완, 리드미, 이미지 제작 함 ... , 라우터 부분 수정했음 (머지하죠)
슬희님 : list page ui 는 다했다, 데이터랑 무한스크롤 (x) (슬라이드) 해야됌 ....

**- 발표 준비하기**
- 노션 꾸미기 (심심하면 하셍, 숨기자!)
- 수영님 소외 노노

**- 배포**
netlify (목요일날 같이해보자!)

**- 오프라인 만남 장소 정하기**
13시 수원 성대

일월 수목원 같이 가는 걸루

발표자료

다같이 성대인인척, 인공지능 쉅

수님 수업 있음 (18시 ~ 19시, 출튀해라! 잡힐뻔 ..)

19시 밥먹으러 맛집을 찾아볼게요

수원에서 윤혁님과 같이 ... 오붓하게

### 💪🏻 오늘 할일

- 개발하기
- 운동하기
- 갓생살기

### 🤔 특이사항

참참참참 데이
11/11 빼빼로 ...

**Comment by oauch** [comment]
벌써 올리시다니.. 너무하십니다(?)

**Comment by ngsy** [comment]
개발하기 운동하기는 할수있지만.. 갓생은 어렵네요😔

**Reply by hanseulhee** [reply]
이미 하고 계세요 !! 😆
